package.json

```diff
@@ -12,19 +12,19 @@
   },
   "private": true,
   "devDependencies": {
-    "@angular-devkit/build-angular": "^14.2.10",
+    "@angular-devkit/build-angular": "^16.2.10",
     "@angular-eslint/builder": "14.4.0",
```

Comment:
**Suggestion** The Angular-eslint packages should all be upgraded to match the angular version